tests/contract_testcases/solana/accessor/accessor_is_external.sol

```diff
@@ -0,0 +1,10 @@
+contract C {
+	int constant public c = 102;
+
+	function f() public {
+		int x = c();
+	}
+}
+// ---- Expect: diagnostics ----
+// error: 5:11-14: functions declared external cannot be called via an internal function call
+// 	note 2:22-23: declaration of function 'c'
```

Comment:
The note points to a public variable, while the error says `functions declared external`. Don't we need a clearer error message for this case?

Reply:
Good idea, I've improved this.